examples/encfs/aci-arm-template.json

```diff
@@ -44,7 +44,7 @@
                   "value": "<optional-loglevel-trace-debug-info-warning-error-fatal-panic>"
                 }
               ],
-              "image": "mcr.microsoft.com/aci/encfs:2.11",
+              "image": "mcr.microsoft.com/aci/encfs:2.12",
```

Comment:
Can you do the same for the ENCFS examples? We made 2.12 for that too. You may also need to update the tests/pipeline versions

Reply:
I thought I did. Where is it 2.11 still?

Reply:
Oops I had a brain fart and misread all these as skr. You're good. Though I would update your pipelines in the other repos to point to the newest versions